cmake/helpers.cmake

```diff
@@ -104,6 +104,12 @@ function(add_umf_target_compile_options name)
                 ${name} PRIVATE -Werror -fno-omit-frame-pointer
                                 -fstack-protector-strong)
         endif()
+        if(USE_GCOV)
+            if(NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
+                message(FATAL_ERROR "To use gcov, the build type must be Debug")
```

Comment:
is it truly a fatal error? I guess we get a different results for Release, but it should work, right...?

Reply:
Soo.. it might work in the build type "RelWithDebInfo"...?